lib/Button/Button.js

```diff
@@ -268,5 +279,16 @@ module.exports = kind(
 		if (!this.showing && this.hasClass('pressed')) {
 			this.undepress();
 		}
+	},
+
+	/**
+	* @private
+	*/
+	backgroundOpacityChanged: function (old) {
+		var opacity = this.get('backgroundOpacity');
```

Comment:
Instead of doing this, can we use the second parameter passed to the change handler i.e. `backgroundOpacityChanged: function (was, is) {`